resources/object_repo/create_dbaasinventory_obj.resource

```diff
@@ -3,11 +3,9 @@ Object Repository for DBaaSInvertories Screen
 
 
 *** Variables ***
-${txtNoDatabaseInstance}        //h2[contains(.,"No Database Instances")]
-${btnCreateProviderAcc}         //button[contains(.,"Create Provider Account")]
-${titleCreateProviderAcc}       //div[@class="section-title" and contains(.,"Create Provider Account")]
+${titleImportProviderAcc}       //div[@class="section-title" and contains(.,"Import Database Provider Account")]
 ${txtInvalidNamespace}          //h4[contains(.,"Invalid Namespace")]
-${btnCreateDisabled}            //button[contains(.,"Create") and @aria-disabled="true"]
```

Comment:
We should have both Create button and Import button as Create button will come into play for creation of DB instances and at other places in future